ansible/roles/miniflux/tasks/main.yml

```diff
@@ -0,0 +1,55 @@
+---
+# Copyright (C) 2021-2022 Berkhan Berkdemir <berkhan@tekdanisman.com>
+# Copyright (C) 2015-2022 DebOps <https://debops.org/>
+# SPDX-License-Identifier: GPL-3.0-only
+
+- name: Import custom Ansible plugins
+  ansible.builtin.import_role:
+    name: 'ansible_plugins'
+
+- name: Import DebOps global handlers
+  ansible.builtin.import_role:
+    name: 'global_handlers'
+
+- name: Import DebOps secret role
+  ansible.builtin.import_role:
+    name: 'secret'
+
+- name: Create required UNIX system group
+  ansible.builtin.group:
+    name: '{{ miniflux__group }}'
+    state: 'present'
+    system: True
+
+- name: Create required UNIX system account
+  ansible.builtin.user:
+    name: '{{ miniflux__user }}'
+    group: '{{ miniflux__group }}'
+    append: True
+    home: '{{ miniflux__home }}'
+    comment: '{{ miniflux__comment }}'
+    shell: '{{ miniflux__shell }}'
+    skeleton: '/dev/null'
+    state: 'present'
+    system: True
+
+- name: Install systemd configuration files
+  ansible.builtin.template:
+    src: '{{ item }}.j2'
+    dest: '/{{ item }}'
+    mode: '0644'
+  loop:
+    - 'etc/systemd/system/miniflux.service'
+    - 'etc/miniflux.conf'
+  notify: [ 'Reload service manager' ]
+
+- name: Run migrations
+  # environment:
+  #   DATABASE_URL: 'postgres://{{ miniflux__database_user }}:{{ miniflux__database_password }}@{{ miniflux__database_host }}:{{ miniflux__database_port }}/{{ miniflux__database_name }}'
+  command: '/usr/local/bin/miniflux -migrate'
```

Comment:
Is this command idempotent? Expect that the role can be executed multiple times against the host.